src/InvertedIndex.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * InvertedIndex class
+ */
+  class InvertedIndex {
+ /**
+  * constructor - contain class properties
+  */
+    constructor() {
+      this.allIndices = {};
+      this.fileContent = '';
+    }
+  /**
+    * createIndex - creates index for json files e.g {of:[0,1,2,3,4]}
+    * @param  {string} fileName : string
+    * @param  {object} fileContent :array of object
+    * @return {object} returns : object
```

Comment:
Instead of returns, what do you think about using the name of the object it returns? Or is 'returns' the name of the object this function reurns?